### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by yoavweiss**
This PR updates the README to be in line with the current proposal

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,30 +104,30 @@ accomplish this as follows:
         Note: See the GREASE-like discussion below for how we could anticipate the inevitable lies
         which user agents might want to tell in this field.
 
-    2.  The `Sec-CH-UA-Platform` header field represents the platform's brand and major version. For example:
+    2.  The `Sec-CH-Platform` header field represents the platform's brand and major version. For example:
```
Comment (mikewest) [inline README.md]: Is this a change we're making? I kinda liked that the `UA-` prefix grouped all these together (and some of them are probably too generically named without it).

Comment (yoavweiss) [reply README.md]: I thought so...
As I see it, we have two options:
Either we align all related FP and Accept-CH strings to say "ua-" as well, or we remove it here. But 3 prefixes is starting to be a lot...

File: README.md [hunk 2]
```diff
@@ -247,34 +242,27 @@ browser on iOS) are interesting cases in which the underlying browser engine on
 doesn't match the engine that the relevant browser built themselves. What should we do in these
 cases?
 
-I think we have a few options for the string:
+We have a few options for the string:
```
Comment (amtunlimited) [inline README.md]: Maybe "There are a few..."? Not sure if it really matters all that much.

Comment (yoavweiss) [reply README.md]: Sure

**Pull request merged**